dockstore-webservice/src/main/java/io/dockstore/webservice/languages/WDLHandler.java

```diff
@@ -435,7 +455,11 @@ public VersionTypeValidation validateTestParameterSet(Set<SourceFile> sourceFile
             // Get import files
             namespaceToPath = wdlBridge.getImportMap(tempMainDescriptor.getAbsolutePath(), mainDescName);
         } catch (WdlParser.SyntaxError ex) {
-            final String exMsg = WDLHandler.WDL_PARSE_ERROR + ex.getMessage();
+            String exMsg = WDLHandler.WDL_PARSE_ERROR + ex.getMessage();
+            Optional<String> semVersionString = getSemanticVersionString(tempMainDescriptor.getAbsolutePath());
+            if (semVersionString.isPresent() && versionIsGreaterThanCurrentlySupported(semVersionString.get())) {
+                exMsg = "WDL version " + semVersionString.get() + " cannot be completely parsed for content at this time.";
```

Comment:
If this message is surfaced in the UI, I suggest a more helpful message. Something like "Dockstore only supports up to WDL 1.0, and you have WDL 1.1" (very rough, needs crafting, but hopefully you get the idea). "at this time" implies to me, that we have a temporary problem on our side.

Reply:
Agreed, "at this time" sounds like we have a temporary outage due to a DDOS or something

Reply:
Fixed up the message and simplified code